vips/foreign.c

```diff
@@ -1,5 +1,6 @@
 #include "lang.h"
 #include "foreign.h"
+#include <stdio.h>
```

Comment:
what's the reason? eliminate compile warnings?

Reply:
I don't know... Not a C master

Reply:
so let's remove it if it is not neccessary

Reply:
done